themes/src/main/resources-community/theme/base/login/messages/messages_es.properties

```diff
@@ -188,13 +263,108 @@ sessionNotActiveMessage=La sesi\u00F3n no est\u00E1 activa
 invalidCodeMessage=Ha ocurrido un error, por favor identificate de nuevo desde tu aplicaci\u00F3n.
 identityProviderUnexpectedErrorMessage=Error no esperado intentado autenticar en el proveedor de identidad.
 identityProviderNotFoundMessage=No se encontr\u00F3 un proveedor de identidad.
+identityProviderLinkSuccess=Se verific\u00F3 con exito su correo electr\u00F3nico. Por favor regrese al navegador y contin\u00FAe con el inicio de sesi\u00F3n.
+staleCodeMessage=esta p\u00E1gina ya no es v\u00E1lida, regrese a su aplicaci\u00F3n e inicie sesi\u00F3n nuevamente
 realmSupportsNoCredentialsMessage=El dominio no soporta ning\u00FAn tipo de credenciales.
+credentialSetupRequired=No se puede iniciar sesi\u00F3n, se requiere la configuraci\u00F3n de credenciales.
 identityProviderNotUniqueMessage=El dominio soporta m\u00FAltiples proveedores de identidad. No se pudo determinar el proveedor de identidad que deber\u00EDa ser utilizado para identificarse.
 emailVerifiedMessage=Tu email ha sido verificado.
+staleEmailVerificationLink=El enlace en el que hizo click est\u00E1 obsoleto, quiz\u00E1 ya haya verificado su correo electr\u00F3nico
+identityProviderAlreadyLinkedMessage=La identidad devuelta por el proveedor de identidad ya est\u00E1 asociada a otro usuario.
+confirmAccountLinking=Confirme la vinculaci\u00F3n de la cuenta {0} del proveedor de identidad {1} con su cuenta
+confirmEmailAddressVerification=Confirme la validez de la direcci\u00F3n de correo electr\u00F3nico {0}.
+confirmExecutionOfActions=Realice la(s) siguiente(s) accion(es)
+
+locale_ca=Catal\u00E0
+locale_de=Deutsch
+locale_en=English
+locale_es=Espa\u00F1ol
+locale_fr=Fran\u00E7ais
+locale_it=Italiano
+locale_ja=\u65E5\u672C\u8A9E
+locale_nl=Nederlands
+locale_no=Norsk
+locale_pl=Polish
+locale_pt_BR=Portugu\u00EAs (Brasil)
+locale_pt-BR=Portugu\u00EAs (Brasil)
+locale_ru=\u0420\u0443\u0441\u0441\u043A\u0438\u0439
+locale_lt=Lietuvi\u0173
+locale_zh-CN=\u4E2D\u6587\u7B80\u4F53
+locale_sk=Sloven\u010Dina
+locale_sv=Svenska
 
 backToApplication=&laquo; Volver a la aplicaci\u00F3n
 missingParameterMessage=Par\u00E1metros que faltan: {0}
 clientNotFoundMessage=Cliente no encontrado
+clientDisabledMessage=Cliente deshabilitado.
 invalidParameterMessage=Par\u00E1metro no v\u00E1lido: {0}
-alreadyLoggedIn=You are already logged in.
+alreadyLoggedIn=Usted ya ha iniciado sesión
```

Comment:
Remove "Usted" and change the "ó"
"Ya ha inicado sesi\u00F3n"

Reply:
Typo in fix :slightly_smiling_face: 
should be "Ya ha iniciado sesi\u00F3n"